### DISCUSSION TITLE
Widgets

### DISCUSSION THREAD

**Discussion by ZachOrr** [the-blue-alliance/the-blue-alliance-ios#909] [Ideas]
Hey y'all - I'm looking for some feedback on what users would want to see in TBA for iOS widgets. Widgets are small views available on iPhone, iPad, and macOS used to elevate content to the Home screen. Keep suggestions mindful of [Apple's Human Interface Guidelines for Widgets](https://developer.apple.com/design/human-interface-guidelines/widgets/overview/introduction/).

My initial thoughts for potential widgets include -
* Previous/Upcoming Match for Team
* Ranking/Record for Team@Event
* Recent/Next 2 Matches for Event

I'd love to hear other ideas for Widget functionality!

**Comment by cyocom** [comment]
- Top Ranks at event (Rank, Team #, # Matches, RP Avg are important fields to me, top 8-12 would be important)
- Latest score breakdown for Team's Previous Match
- Latest score breakdown for Event's Previous Match
- +1 for Upcoming Matches for Team (probably next 2 with Scheduled/Estimated Time)
- +1 for Upcoming Matches for Event (w/ Favorite Team highlights)

**Reply by ZachOrr** [reply]
I love the idea of Upcoming Matches for an Event with a filter/highlight for myTBA favorites!

**Comment by ZachOrr** [comment]
Pending GameDay for iOS, we might be able to show a Twitch-style widget that surfaces live special webcasts/live events. We're uniquely positioned to show this data due to our webcast aggregation.

**Comment by ZachOrr** [comment]
Dropping some design inspiration here

<img width="631" alt="Screen Shot 2021-06-08 at 2 51 29 PM" src="https://user-images.githubusercontent.com/516458/122272468-971fb100-ceae-11eb-8f7f-e3cf8d00819c.png">
